src/dev/app.md

```diff
@@ -29,15 +31,63 @@ docker pull cozy/cozy-app-dev
 
 (We update this image on a regular basis with the latest version of the server and our library. Don’t forget to update the image by running `docker pull cozy/cozy-app-dev` from time to time).
 
+## Create your application
 
-## Create your first application
+You can boostrap your application from scratch if you want, but we recommand to use our new community tool [`create-cozy-app`](https://github.com/CPatchane/create-cozy-app) to bootstrap very easily a Cozy application for you.
+
+This tool will generate an application using (P)React, the framework we internally use in the Cozy Front team. But [options](https://github.com/CPatchane/create-cozy-app#options) are available if you want to use other frameworks.
+
+!!! warning ""
+    For now the new [`cozy-client`](https://github.com/cozy/cozy-client) is used only in the (P)React template (it doesn't use the previous `cozy-client-js` anymore). This library is at an early stage but you can use it, it will be our next Cozy client for application development.
+
+First of all, run directly `create-cozy-app` without installing it globally by using the `yarn create cozy-app` command to bootstrap your application:
+
+```
+yarn create cozy-app mycozyapp
+```
+
+The script will download some dependencies (may take a while) and ask you a few questions, then create an application skeleton inside `mycozyapp`.
+
+That's all! You can start hacking:
+
+```
+cd mycozyapp
+yarn watch:standalone
+```
+
+After the webpack build, your app should be available at http://localhost:8888
+
+> You can change the host and the port of your application server here by using respectively the environment variables HOST and PORT
+
+### Run it inside a Cozy using Docker
+
+You can run your application (here `mycozyapp`) inside a Cozy thanks to the [cozy-stack docker image][cozy-stack-docker]:
+
+```sh
+# in a terminal, run your app in watch mode
+$ cd mycozyapp
+$ yarn watch:browser
+```
+
+Then, in another terminal:
+
+```sh
+# in another terminal, run the docker container
+$ yarn stack:docker
+# or if you want the complete command (see more documentation below)
+$ docker run --rm -it -p 8080:8080 -v "$(pwd)/build":/data/cozy-app/mycozyapp cozy/cozy-app-dev
+```
+
+Your app is now available at http://mycozyapp.cozy.tools:8080.
```

Comment:
Again, docker should not appear in the documentation except in a small insert like this: trick, you can use docker if you are more comfortable with it.